frontend/src/components/Dropdown/MultiSelect/MultiSelectProvider.tsx

```diff
@@ -252,6 +271,13 @@ export const MultiSelectProvider = ({
     isEmpty: selectedItems.length === 0,
   })
 
+  const virtualListHeight = useMemo(() => {
+    const totalHeight = filteredItems.length * 48
```

Comment:
hardcoding this 48 causes issues in the case where there are a small number of options that are very long (i.e. when `filteredItems.length` is small => `totalHeight` is small => `virtualListHeight` is small, but the length of the options stretch over multiple rows), case in point

normal usage:
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/25571626/177072434-6d7fcc58-42af-47b9-af1a-6ed37d87ff8e.png">

long option but not many of them
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/25571626/177072525-0de9949d-24d4-4722-9469-5a8fc02d33ae.png">

in this case it seems that we would like actually for `virtualListHeight` to be the `VIRTUAL_LIST_MAX_HEIGHT` (or at least, bigger than what it shows currently) 

One option to fix this is not to calculate the `virtualListHeight` here, but rather in `SelectMenu` and `MultiSelectMenu` itself by getting the actual heights of the `DropdownItems`?

https://stackoverflow.com/questions/35153599/reactjs-get-height-of-an-element

Reply:
conscious decision for performance. this is good enough for 99% of the cases tbh, and we do not need to do intensive calculations of dom height for dynamicism.

Reply:
fair point, just wanted to make sure we considered that case!